### PR TITLE
No encrypted password and README.md update

### PR DESCRIPTION
**Pull request by patrykdros97**

### CODE REVIEW

File: fdse_soar_installer.yaml [hunk 1]
```diff
@@ -88,20 +88,27 @@
...
+        name: passlib
+        executable: pip3
 
     - name: Download phantom tar package
       become: yes
       become_user: soaruser
       ansible.builtin.get_url:
         url: '{{ vars.soar_download_url }}'
-        dest: /tmp/splunk-soar.tar
+        dest: /tmp/phantom.zip
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:104]: please leave this as it was or as .tar.gz (depending on what the actual file is). .zip works the same but is a misleading name :)

File: fdse_soar_installer.yaml [hunk 2]
```diff
@@ -88,20 +88,27 @@
...
 
     - name: extract phantom tar package
       become: yes
       become_user: soaruser
       ansible.builtin.unarchive:
         remote_src: true
-        src: /tmp/splunk-soar.tar
+        src: /tmp/phantom.zip
```
Comment (ricardom-splunk) [inline fdse_soar_installer.yaml:111]: same as above

**Review by ricardom-splunk — CHANGES_REQUESTED** [review]
As discussed in Slack, requesting you to make a couple of changes :)

**Comment by patrykdros97** [comment]
Corrected as you suggested

**Pull request merged**